tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/CatalogConstants.java

```diff
@@ -48,11 +48,16 @@
   public static final String TB_STATISTICS = "STATS";
   public static final String TB_PARTITION_METHODS = "PARTITION_METHODS";
   public static final String TB_PARTTIONS = "PARTITIONS";
+  public static final String TB_PARTTION_KEYS = "PARTITION_KEYS";
 
   public static final String COL_TABLESPACE_PK = "SPACE_ID";
   public static final String COL_DATABASES_PK = "DB_ID";
   public static final String COL_TABLES_PK = "TID";
   public static final String COL_TABLES_NAME = "TABLE_NAME";
+
+  public static final String COL_PARTITIONS_PK = "PID";
```

Comment:
I followed existing naming rule of CatalogConstants. I think there are three options as follows.
 - COL_PARTITIONS_PK
 - COL_PARTITIONS_ID
 - PARTITIONS_ID

What's your opinion about above options? :)